### PR TITLE
update to angular 15

### PR DESCRIPTION
**Pull request by bruno-garcia**
Seems like we have both package-lock and yarn.lock in use, I'll remove package-lock

### CODE REVIEW

File: src/NuGetTrends.Web/Portal/src/styles.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* Font Awesome */
 $fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome.scss";
+// @import "~font-awesome/scss/font-awesome.scss";
```
Comment (joaopgrassi) [inline src/NuGetTrends.Web/Portal/src/styles.scss]:
```suggestion
@import "node_modules/font-awesome/scss/font-awesome.scss";
```

File: src/NuGetTrends.Web/Portal/src/styles.scss [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 /* Bulma Color Overrides*/
 @import './colors';
-@import '~bulma/bulma.sass';
+// @import '~bulma/bulma.sass';
```
Comment (joaopgrassi) [inline src/NuGetTrends.Web/Portal/src/styles.scss]:
```suggestion
@import 'node_modules/bulma/bulma.sass';
```
I quickly tested and this works (need to test better with the prod build still). If we remove this the site is all broken :D. I guess Angular did some magic with the path/bundles before.. :thinking:

**Review by joaopgrassi — COMMENTED** [review]
I tried the branch locally and for me the tests don't run anymore.. maybe it's some issue with my setup.

I also saw there's some deprecated code for the tests, and a bunch of others follow ups that I'm not sure we use so need to check (https://update.angular.io/?v=13.0-15.0).

I will try to follow up on those. OK if I send some commits to this branch?

File: src/NuGetTrends.Web/Portal/src/styles.scss (hunk 2 above)
Comment (bruno-garcia) [reply src/NuGetTrends.Web/Portal/src/styles.scss]: dang I wish I read this before, just figured this out after some struggling

**Comment by bruno-garcia** [comment]
> I will try to follow up on those. OK if I send some commits to this branch?

Sure thing! Thanks.

File: src/NuGetTrends.Web/Portal/src/app/app.module.ts [hunk 3]
```diff
@@ -18,13 +19,21 @@ import { CoreModule } from './core/core.module';
...
   integrations: [
-    new Integrations.BrowserTracing({
-      routingInstrumentation: Sentry.routingInstrumentation,
+    new Replay({
+      // No PII here so lets get the texts
+      maskAllText: false,
+      blockAllMedia: true,
+    }),
+    new BrowserTracing({
+      routingInstrumentation: Sentry.instrumentAngularRouting,
```
Comment (bruno-garcia) [inline src/NuGetTrends.Web/Portal/src/app/app.module.ts:34]: https://github.com/getsentry/sentry-javascript/blob/2fa11eb54f13b305dbee35687539748614583d6e/packages/angular/src/index.ts#L9

**Comment by joaopgrassi** [comment]
Build seems to pass now. I need to setup the contrib db locally as I purged all my containers a while ago :sweat_smile: so I can try the prod build with some data, as chartjs was also updated. It should all be fine but will double check.

**Comment by bruno-garcia** [comment]
Nice! Thanks Joao

**Review by joaopgrassi — APPROVED** [review]
Thanks!

**Pull request merged**